### PR TITLE
refactor(zupass-gatekeeper): improved gatekeeper script

### PR DESCRIPTION
**Pull request by crisgarner**
Improved gatekeeper script to avoid doing conversion on deploy config

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

**Review by ctrlc03 — APPROVED** [review]
Thanks

File: packages/contracts/tasks/deploy/maci/02-gatekeepers.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import { hexToBigInt, uuidToBigInt } from "@pcd/util";
+
 import { HatsGatekeeperBase } from "../../../typechain-types";
 import { EDeploySteps, ESupportedChains } from "../../helpers/constants";
 import { ContractStorage } from "../../helpers/ContractStorage";
 import { Deployment } from "../../helpers/Deployment";
-import { uuidToBigInt } from "../../helpers/numericParser";
```
Comment (0xmad) [inline packages/contracts/tasks/deploy/maci/02-gatekeepers.ts:5]: If we don't use it anymore, let's remove the functions inside the file

Comment (crisgarner) [reply packages/contracts/tasks/deploy/maci/02-gatekeepers.ts:5]: Thanks! fixed

**Comment by ctrlc03** [comment]
@crisgarner could you rebase this please?

**Pull request merged**